frontend/src/api_methods.ts

```diff
@@ -69,3 +76,11 @@ export function getSpotifyAccessTokenFromBackend(spotifyCode: string) {
     return axios.get("/api/spotify/callback" + spotifyCode)
         .then((response:AxiosResponse<SpotifyLoginResponseDTO>) => response.data)
 }
+
+export function getAllUserPlaylists() {
+    return axios.get("api/soundtracker/user-favorites/spotify-playlists/" + localStorage.getItem("spotify_jwt"), {
+        headers: {
+            Authorization: "Bearer " + localStorage.getItem("jwt")
+        }
+    }).then((response:AxiosResponse<SpotifyUserPlaylists>) => response.data)
+}
```

Comment:
Here you could define a `requestConfig` on the start of the file like so: 
```javascript
let requestConfig = {
   headers: { 
      Authorization: `Bearer ${localStorage.getItem("jwt")`
   }
```

and then you could just use thie variable everytime you need it like so: 

````Javascript
return axios.get("api/soundtracker/user-favorites/spotify-playlists/" + localStorage.getItem("spotify_jwt"), requestConfig).then(()=>{
   //something...
})
```

Reply:
what i sayed was not wrong, but its not working like intended. If you solve this like i showed ypu here ypu will have to reload the page if u make an request. 

But this can be prevented like so: 

```Javascript
const requestConfig = () => {
   return {
      headers: { 
         Authorization: `Bearer ${localStorage.getItem("jwt")`
   }
}
```
And then in your request function: 

```Javascript
return axios.get("api/soundtracker/user-favorites/spotify-playlist/" + localStorage.getItem("spotify_jwt"), requestconfig())
```